nixos/modules/services/misc/gitlab.nix

```diff
@@ -159,6 +159,7 @@ let
         api_url = "http://${config.services.dockerRegistry.listenAddress}:${toString config.services.dockerRegistry.port}/";
         issuer = cfg.registry.issuer;
       };
+      elasticsearch.indexer_path = "${pkgs.gitlab-elasticsearch-indexer}/bin/gitlab-elasticsearch-indexer";
```

Comment:
this unconditionnaly adds it to closure, cant it be passed via extraConfig instead ? or maybe it warrants a dedicated option